src/util/sexpr.cpp

```diff
@@ -279,7 +279,7 @@ bool SExpr::languageQuotesKeywords(OutputLanguage language) {
       return true;
     case language::output::LANG_AST:
     case language::output::LANG_CVC3:
-    case language::output::LANG_CVC4:
+    case language::output::LANG_CVC:
```

Comment:
Not sure if this should really be renamed. @ajreynol  what do you think?

Reply:
Do we even still support LANG_CVC3 @ajreynol ?

Reply:
I think LANG_CVC3 can be deleted, it doesn't appear to be used.

Reply:
Resolved offline, yes we can rename and yes we can remove LANG_CVC3. Will do the latter in a separate PR.